public/bower.json

```diff
@@ -2,15 +2,15 @@
   "name": "public",
   "version": "0.0.0",
   "dependencies": {
-    "angular": "1.2.15",
+    "angular": "1.4.14",
```

Comment:
wow. I put this together a _looong_ time ago.

Reply:
I'm going to try to get it upgraded to 1.8.x too after I solve the NaN bug and the configuration-watching bug in angular-dialgauge.